Source/WTF/wtf/EnumeratedArray.h

```diff
@@ -43,14 +43,7 @@ class EnumeratedArray {
     using const_reference = const value_type&;
     using pointer = value_type*;
     using const_pointer = const value_type*;
-
-private:
-    // We add 1 to the size because we expect that LastValue is the maximum value of the enum,
-    // rather than the count of items in the enum.
-    // We're assuming the values in the enum are zero-indexed.
```

Comment:
Is this comment not useful?

Reply:
The fact that we assume the values are zero-based, and that LastValue is indeed the last value and not one past the last value, are already mentioned in the comment at the top of the class, a few lines up. I think the explanation of why we need to add 1 is reasonably self-evident, but I'm happy to leave it in if you prefer.

Reply:
Up to you. I'm the author of that comment so I think I'm biased :P